guides/source/active_record_multiple_databases.md

```diff
@@ -247,13 +247,13 @@ The `database` argument for `connected_to` will take a symbol or a config hash.
 
 Note that `connected_to` with a role will look up an existing connection and switch
 using the connection specification name. This means that if you pass an unknown role
-like `connected_to(role: :nonexistent)` you will get an error like that says
+like `connected_to(role: :nonexistent)` you will get an error that says
```

Comment:
Should be removed this "like"?

Reply:
@kamipo 

`like` seems inappropriate at the place it was used. It needs to be associated with the error message that follows.

How about rewording as,
```diff
- you will get an error like that says `ActiveRecord::ConnectionNotEstablished (No connection pool with 'AnimalsBase' found for the 'nonexistent' role.)`
+ you will get an error that says something like `ActiveRecord::ConnectionNotEstablished (No connection pool with 'AnimalsBase' found for the 'nonexistent' role.)` 
```
instead of removing `like` word?

Reply:
lgtm